apigw-websocket-mapping-template-authorizer/src/onconnect.mjs

```diff
@@ -0,0 +1,22 @@
+        import { DynamoDBClient, PutItemCommand } from "@aws-sdk/client-dynamodb";
+        const client = new DynamoDBClient({ region: process.env.AWS_REGION });
+        export const handler = async (event) => {
+          const connectionId = event.requestContext.connectionId;
+          console.log("connection ID:", connectionId ); // Print the connection ID
```

Comment:
CWE-117 - Log injection High
We detected that unsanitized user data is being written to the logs. Unsanitized data can inject malicious content. To increase the security of your code, sanitize your inputs before logging them.